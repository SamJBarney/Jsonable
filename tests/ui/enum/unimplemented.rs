use jsonable::*;

#[derive(Jsonable)]
enum Enum {
    ASDF
}

fn main()  {}