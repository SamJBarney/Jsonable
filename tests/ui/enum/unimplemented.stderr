error: proc-macro derive panicked
 --> tests/ui/enum/unimplemented.rs:3:10
  |
3 | #[derive(Jsonable)]
  |          ^^^^^^^^
  |
  = help: message: Unimplemented
